static/app/views/starfish/queries/useSpanSamples.tsx

```diff
@@ -54,6 +71,10 @@ export const useSpanSamples = (options: Options) => {
         `${url}?${qs.stringify({
           ...dateCondtions,
           ...{utc: location.query.utc},
+          lowerBound: 0,
+          firstBound: (maxYValue * 1) / 3,
+          secondBound: (maxYValue * 2) / 3,
```

Comment:
nit: why not 
```suggestion
          firstBound: maxYValue * (1 / 3),
          secondBound: maxYValue * (2 / 3),
```

Reply:
Yeah i like that better too 👍 .

So originally wrote this as
<img width="305" alt="image" src="https://github.com/getsentry/sentry/assets/44422760/76746fbc-b2bc-44e3-9888-ce4b44b967c1">
But prettier autoformatted it to 
```firstBound: (maxYValue * 1) / 3```

Prettier seems cool with 
```firstBound: maxYValue * (1 / 3), ```
as well, so i'll update it